### PR TITLE
Feature/otc300 consume commit message

### PR DESCRIPTION
**Pull request by bothadeon**
Removed the unit test code on the Facade. To be done in a separate story.
Rest of the unit test are at 100% coverage for the API

### CODE REVIEW

**Comment by ndonnan** [comment]
I noticed that the files, functions etc are named 'transferFulfilxxx'. I think it would be more clear what they do if they were named 'transferParticipantStatexxx' because this isn't dealing with transfer fulfilment but rather updating the transferParticipantState, what do you think?

**Comment by bothadeon** [comment]
Yip , makes sense

**Comment by bothadeon** [comment]
Current code coverage is at :
=============================== Coverage summary ===============================
Statements   : 98.2% ( 1421/1447 )
Branches     : 99.23% ( 387/390 )
Functions    : 95% ( 190/200 )
Lines        : 98.15% ( 1380/1406 )
================================================================================

The coverage for model/transferFulfil facade.js needs to be investigated further seeing that Sinon seems to have difficulty mocking nested functions more than three levels deep.

A story will be created to specifically look into this

File: config/default.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "DATABASE": {
     "DIALECT": "mysql",
     "HOST": "localhost",
-    "PORT": 3306,
```
Comment (rmothilal) [inline config/default.json:7]: please revert to 3306

Comment (bothadeon) [reply config/default.json]: Port reverted to 3306

**Comment by bothadeon** [comment]
The unit test on the models/transferParticipanttStateChange/Facade.js has been been ignored for now until the nested function issue in Sinon is resolved. All the other models meet the 90% or more code coverage criteria

**Comment by bothadeon** [comment]
The current code coverage (Without Facade.test.js are as follows :
1..635
# tests 635
# pass  635

# ok


=============================== Coverage summary ===============================
Statements   : 99.93% ( 1427/1428 )
Branches     : 99.74% ( 389/390 )
Functions    : 100% ( 192/192 )
Lines        : 99.93% ( 1386/1387 )
================================================================================

**Pull request merged**